modules/rest-api/pages/rest-configure-saml.adoc

```diff
@@ -0,0 +1,205 @@
+
+|spMetadataCacheDuration
+|Sets how long the IdP caches Couchbase Server's metadata. This value is in https://en.wikipedia.org/wiki/ISO_8601#Durations[ISO8601 duration format^]. The default value `P1M` sets this duration to one month.
+
+|spMetadataURL
+|The URL to retrieve Couchbase Server's SAML metadata. By default, Couchbase Server sets this value based on spBaseURLType and spBaseURLScheme.
+
+|spOrgDisplayName
+|An alternate name of your organization. The IdP can use this value for various purposes, such as when prompting a user to authenticate.
+
+|spOrgName
+|The name of your organization. The IdP can use this value for various purposes, such as when prompting a user to authenticate.
+
+|spOrgURL
+|A URL for your organization. This value does not play a role in authentication. The IdP can use this value for various purposes, such as when prompting a user to authenticate.
+
+|spSessionExpire
+|String value that controls whether the Web Console expires its session when the IdP's authorization expires. When set to the default `SessionNotOnOrAfter` value, the Web Console has the user reauthenticate when the IdP's authentication expires. If set to `false`, the Web Console does not set its session length to match the IdP's authentication expiration.
+
+|spSignMetadata
+|A Boolean value that controls whether Couchbase Server signs its metadata using the key set in spKey. Defaults to `true`, which enables signing the metadata.
+
+|spSignRequests
+|A Boolean value that controls whether Couchbase Server signs its SAML requests to the IdP. Defaults to `true` which enables signing the requests.
+
+|spTrustedFingerprints
+|One or more certificate fingerprints that Couchbase Server can use to authenticate messages from the IdP instead of using certificates. 
+
+|spTrustedFingerprintsUsage
+|String value that controls what Couchbase Server verifies using certificate fingerprints. When set to the default `metadataInitialOnly` value, it uses fingerprints to verify just the metadata it retrieves from the IdP. If set to `everything` it uses fingerprints for all SAML message verification. Only has an effect if at least one fingerprint is set in spTrustedFingerprints.
+
+|spVerifyAssertionEnvelopSig
+|A Boolean value. When set to `true`, Couchbase Server verifies the IdP's signature on the entire SAML message envelope. Only has an effect if idpMetadataTLSCAs contains the IdP's certificate. 
+
+|spVerifyAssertionSig
+|A Boolean value. When set to the default `true` value, Couchbase Server verifies the IdP's signature on the assertion portion of SAML messages. Only has an effect if idpMetadataTLSCAs contains the IdP's certificate. 
+
+|spVerifyLogoutReqSig
+|A Boolean value. When set to `true`, Couchbase Server verifies the IdP's signature on SAML logout requests. Only has an effect if idpMetadataTLSCAs contains the IdP's certificate. 
```

Comment:
Please remove "Only has an effect if idpMetadataTLSCAs contains the IdP's certificate. "